roles/obol_lodestar/templates/zypher/mainnet/systemd.service.j2

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description= Obol Lodestar Service
+
+[Service]
+Type=simple
+User={{ service_identifier }}
+Group={{ service_identifier }}
+WorkingDirectory={{ obol_lodestar_bin_dir }}
+ExecStart={{ obol_lodestar_bin_dir }}/{{ service_identifier }} validator \
+    --dataDir {{ obol_lodestar_data_dir }} \
+    --keystoresDir {{ obol_lodestar_key_dir }} \
+    --secretsDir {{ obol_lodestar_key_dir }} \
```

Comment:
are these flag present ? I was not able to find it in ./obol-lodestar validator --help

Reply:
also there are being pointed to same directory ?